src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineNameModal/EditBaselineNameModal.js

```diff
@@ -62,14 +62,14 @@ export class EditBaselineNameModal extends Component {
                     isRequired
                     fieldId='baseline name'
                     helperTextInvalid={ error.hasOwnProperty('detail') ? error.detail : null }
-                    isValid={ !error.hasOwnProperty('status') }
+                    validated={ error.hasOwnProperty('status') ? 'error' : null }
```

Comment:
You could use `ValidatedOptions.error` here as well, and move this into a `const` to use here and on the `TextInput`.